src/js/Content/Features/Community/Inventory/FInventoryMarketHelper.js

```diff
@@ -152,70 +152,88 @@ export default class FInventoryMarketHelper extends Feature {
         }
     }
 
-    _setBackgroundOption(thisItem, assetId, itemActions) {
+    async _setBackgroundOption(thisItem, itemActions) {
 
-        if (!document.querySelector(".inventory_links")) { return; }
         if (itemActions.querySelector(".es_set_background")) { return; }
 
+        // Make sure the selected item is a background
         const viewFullBtn = itemActions.querySelector("a");
-        if (!viewFullBtn) { return; }
+        if (!viewFullBtn || !/public\/images\/items/.test(viewFullBtn.href)) { return; }
```

Comment:
```suggestion
        if (!viewFullBtn || !viewFullBtn.href.includes("public/images/items")) { return; }
```

Reply:
Fixed along with removing the extension part from the regex to match background urls.